lib/tasks/build.js

```diff
@@ -61,5 +62,9 @@ module.exports = Task.extend({
 
         throw err;
       });
+  },
+
+  onInterrupt: function() {
+    return RSVP.Promise.reject(1);
```

Comment:
Looks like `onInterrupt` should be able just to return an exit code without a promise if no async work is necessary